apps/site/tests/e2e/general-behavior.spec.ts

```diff
@@ -70,11 +73,22 @@ test.describe('Node.js Website', () => {
       await expect(themeToggle).toBeVisible();
 
       const initialTheme = await getTheme(page);
+      const initialAriaLabel = await themeToggle.getAttribute('aria-label');
+      expect(initialAriaLabel).toBe(
+        englishLocale.components.common.themeToggle[initialTheme]
```

Comment:
Can we move this to locators?

i.e. `themeToggle { dark, light }`, just so it's easier to adjust in the future.